Homework_3/task_2.py

```diff
@@ -0,0 +1,14 @@
+# Дан список повторяющихся элементов.
+# Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+lst = [1, 1, 1, 2, 3, 4, 4, 5, 6, 7, 8, 8, "asd", "qwe", "asd"]
+ONE = 1
+duplicate_elements_lst = []
+print(f'\nThis list: \n{lst}')
+for item in lst:
```

Comment:
Можно немного понизить сложность делав так
```suggestion
for item in set(lst):
```